### DISCUSSION TITLE
Is it safe to mix & match either OS platforms or mitmproxy versions for record/replay?

### DISCUSSION THREAD

**Discussion by 92kns** [mitmproxy/mitmproxy#5566] [Q&A] [answered]
Hello! 
I had a couple of questions on the usage of mitmproxy when I am examining web page loading.

1 - Is it ok if I record website activity on some platform (say linux) and replay the recording on a different platform? (windows or macOS)? 

2 - Is there any concern in replaying recordings done in an older mitmproxy version with a newer mitmproxy version? e.g. recorded with 6, and replay with 8. 

For both cases, it does work when I mix/match version & platforms like this, but i want to be sure that this is safe to do so.

As for what I am doing with these recordings/replays: I am collecting some of the metrics related to page loads as detailed here [sitespeed.io](https://www.sitespeed.io/documentation/sitespeed.io/metrics/)

So my main concern is, would this 'mixing and matching' affect these types of results? 

Thanks!

**Comment by mhils** [comment, marked as answer]
1. Yes, perfectly fine.
2. No concerns. The other way around won't work (recordings are only forward-compatible), but it doesn't sound like that's an issue here.

**Reply by 92kns** [reply]
Great, thanks for confirming @mhils !
